src/audio/Kconfig

```diff
@@ -33,6 +33,15 @@ config COMP_CHAIN_DMA
 	  help
 	    Chain DMA support in hardware
 
+config COMP_IPCGTW
+        bool "IPC Gateway"
+        default y
+        depends on IPC_MAJOR_4
+        help
+          Select for IPC Gateway. IPC Gateway is used to transfer audio between
+          host and DSP via memory window (audio payload) and IPC messages
+          (set/get/flush commands).
```

Comment:
and I would also make it clear if this means that DMA is not used as I think NXP use a similar solution for IPC3.

Reply:
Updated in new version.